### PR TITLE
feat(pipeline-ui): batch export icons

### PR DESCRIPTION
**Pull request by tiina303**
## Problem

Batch exports didn't have an icon, picked the icons from the old apps repos

## Changes

👉 *Stay up-to-date with [PostHog coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review.*

### CODE REVIEW

**Review by Twixes — CHANGES_REQUESTED** [review]

File: frontend/src/scenes/pipeline/destinationsLogic.tsx [hunk 1]
```diff
@@ -44,9 +45,10 @@ export enum PipelineAppBackend {
     Plugin = 'plugin',
 }
 
-export interface BatchExportDestination extends DestinationTypeBase {
+interface BatchExportDestinationType extends DestinationTypeBase {
     backend: PipelineAppBackend.BatchExport
     id: string
+    destination_type: BatchExportDestination['type']
```
[the remark this answers is not in the document]
Comment (tiina303) [reply frontend/src/scenes/pipeline/destinationsLogic.tsx:51]: chatGPT suggested `data storage type` I feel like service would be confusing too

File: frontend/src/scenes/pipeline/icons/Redshift.png
[the remark this answers is not in the document]
Comment (tiina303) [reply frontend/src/scenes/pipeline/icons/Redshift.png:1]: I got them both originally from our plugin repos, not sure where to best get these icons, so yes they look different now too 😅  but at least look good both black and white background now

**Comment by tiina303** [comment]
quick tips for icon location problem? do they need to be in the 'public' folder?

**Review by Twixes — APPROVED** [review]
Looks good, I think the location makes sense

**Comment by Twixes** [comment]
Ah, right, only Storybook's Webpack config wasn't happy. `public` it is.

**Pull request merged**